src/main/java/org/fairdatapoint/service/boostrap/components/MetadataRecordsBootstrapper.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * The MIT License
+ * Copyright © 2016-2024 FAIR Data Team
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.fairdatapoint.service.boostrap.components;
+
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.rio.Rio;
+import org.fairdatapoint.database.rdf.repository.RepositoryMode;
+import org.fairdatapoint.database.rdf.repository.generic.GenericMetadataRepository;
+import org.fairdatapoint.service.boostrap.BootstrapContext;
+import org.fairdatapoint.service.boostrap.fixtures.RecordsFixture;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Locale;
+
+import static org.fairdatapoint.util.ValueFactoryHelper.i;
+
+@Slf4j
+@Component
+public class MetadataRecordsBootstrapper extends AbstractBootstrapper {
+    private final GenericMetadataRepository genericMetadataRepository;
```

Comment:
This makes the `MetadataRecordsBootstrapper` special, because it's the only one that writes to the triple store instead of the relational db. I think this should be made explicit somehow. At the very least a clear explanation in javadoc would be very helpful.